### ISSUE TITLE
Incorrect jsx line comment of attribute 

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/vscode#36175]
From https://github.com/Microsoft/vscode/issues/36097

1. For the jsx

```js
<div
  className="foo"
/>
```

2. Toggle a line comment on the line `className`

**Expected**

```js
<div
  // className="foo"
/>
```

**Actual**

```js
<div
  {/* className="foo baz" */}
/>
```

**Comment by ramya-rao-a** [comment]
I still see
```
<div
  {/* className="foo baz" */}
/>
```

**Comment by mjbvz** [comment]
@ramya-rao-a Do you have any  custom grammars installed? I can't repo this:

![nov-02-2017 11-39-45](https://user-images.githubusercontent.com/12821956/32344009-9be89890-bfc2-11e7-8fd8-4015bfcc2701.gif)

**Comment by ramya-rao-a** [comment]
That's right.
Without the extension, the fix works

**Comment by OliverJAsh** [comment]
Which extension is causing this? None of mine are obvious candidates. I can still repro this in 1.17.2.

**Comment by mjbvz** [comment]
@OliverJAsh This is only fixed in insiders / VS Code 1.18
